lib/client/api.go

```diff
@@ -1047,9 +1047,25 @@ func (tc *TeleportClient) ConnectToProxy() (*ProxyClient, error) {
 	return proxyClient, nil
 }
 
-// Logout locates a certificate stored for a given proxy and deletes it
+// Logout removes certificate and key for the currently logged in user from
+// the filesystem and agent.
 func (tc *TeleportClient) Logout() error {
-	return trace.Wrap(tc.localAgent.DeleteKey())
+	err := tc.localAgent.DeleteKey()
```

Comment:
`trace.Wrap(nil)` returns `nil` so this change is not necessary

Reply:
This makes code substantially more readable. I can't count then umber of times I've skipped over these one liners when reading code only to have to come back to them.